rnd/autogpt_server/autogpt_server/app.py

```diff
@@ -1,61 +1,44 @@
-from multiprocessing import freeze_support, set_start_method
 from typing import TYPE_CHECKING
 
-import Pyro5.api as pyro
-from tenacity import retry, stop_after_attempt, wait_exponential
-
 from .util.logging import configure_logging
 
 if TYPE_CHECKING:
     from autogpt_server.util.process import AppProcess
 
 
-@retry(stop=stop_after_attempt(30), wait=wait_exponential(multiplier=1, min=1, max=30))
-def wait_for_nameserver():
```

Comment:
were you having an issue with this locally?

Reply:
I'm having a problem to call this websocket server only using this function, so I moved it on thr right place (def health check). The execution order will still be exactly the same.

Reply:
I can't do `run_processes(websocketserver())` or anything other than pyronameserver as the first entry.

This change is simply moving the `wait_for_ns` to block after calling the .start(background=True) on the. pyronameserver's part of the code, not here.

Reply:
okay but i think we need to test this before merging as it's caused issues before

Reply:
Btw this doesn't change anything at all, it's just the placing of the code.